tests/system/tostring.nim

```diff
@@ -1,120 +0,0 @@
-discard """
```

Comment:
This shouldn't be removed. I guess you've had some troubles with the PR? Maybe re-create?